cmake/zlib.cmake

```diff
@@ -37,6 +37,8 @@ MACRO (MYSQL_CHECK_ZLIB_WITH_COMPRESS)
   IF(WITH_ZLIB STREQUAL "bundled")
     MYSQL_USE_BUNDLED_ZLIB()
   ELSE()
+    FIND_PACKAGE(PkgConfig QUIET)
+    PKG_GET_VARIABLE(ZLIB_ROOT zlib prefix)
     INCLUDE(FindZLIB)
```

Comment:
odd, but not your fault, not sure why this isn't `find_package(zlib)`

Reply:
Thank you for pointing out, I can also change the line to use `find_package()` which is more conscience

Reply:
What happens if zlib is not installed. What is the value of ZLIB_ROOT?